app/views/job_invocations/_tab_overview.html.erb

```diff
@@ -1,59 +1,29 @@
-<div id="status_chart">
-  <%= job_invocation_chart(job_invocation) %>
-</div>
-
-<div class="col-md-6 infoblock">
-  <h4><%= _('Target hosts') %></h4>
-  <% if job_invocation.bookmark.present? %>
-    <%= _('Bookmark') %> <%= job_invocation.bookmark.name %><br>
-  <% else %>
-    <%= _('Manual selection') %>
-  <% end %>
-  <%= 'using ' + _(Targeting::TYPES[job_invocation.targeting.targeting_type]) %>
-  <pre><%= job_invocation.targeting.search_query %></pre>
-
-  <%= _('Evaluated at:') %> <%= job_invocation.targeting.resolved_at %><br>
-  <% if job_invocation.pattern_template_invocations.size > 1 %>
-    <% job_invocation.pattern_template_invocations.each do |template_invocation| %>
-      <%= host_counter template_invocation.template.provider.humanized_name, ForemanTasks::Task::DynflowTask.for_action(Actions::RemoteExecution::RunHostJob).for_resource(template_invocation).uniq.size %>
-    <% end %>
-  <% end %>
-  <%= host_counter(_('Total hosts'), job_invocation.total_hosts_count) %>
-</div>
-
-<div class="col-md-6 infoblock">
-  <h4><%= _('Providers and templates') %></h4>
-  <% job_invocation.pattern_template_invocations.each do |template_invocation| %>
-    <h5>
-      <b><%= template_invocation.template.name %></b> <%= 'through' %> <%= template_invocation.template.provider.humanized_name %>
-    </h5>
-    <% target = template_invocation.targeting.hosts.with_os.first || template_invocation.targeting.hosts.first %>
-    <%= _('Preview for target %s') % target.try(:name) || 'N/A' %>
-
-    <%= preview_box(template_invocation, target) %>
-
+<% template_invocations = job_invocation.pattern_template_invocations %>
```

Comment:
This should go away alongside support for a JobInvocation for multiple providers at the same time IMO